app/league/dashboard/models.py

```diff
@@ -262,6 +272,8 @@ def episode_stats(cls, episode=None, season=None, num_players=5):
                 kyus_killed[game.black.id] = \
                     kyus_killed.get(game.black.id, 0) + 1
 
+        win_ratios = {p.id: wins[p.id] / games_played[p.id] for p in players}
```

Comment:
though, it doesn't seem to mind: 
![image](https://user-images.githubusercontent.com/3718321/29051286-4c75db6c-7b96-11e7-9c45-aab61d67bff9.png)